.github/workflows/SwiftLint.yml

```diff
@@ -44,7 +44,7 @@ jobs:
       - uses: microsoft/setup-msbuild@v1.3.1
 
       - name: package
-        run: msbuild ${{ github.workspace }}/SourceCache/swift-build/platforms/Windows/SwiftLint.wixproj -nologo -restore -p:Configuration=Release -p:RedistributablesDirectory="${env:SDKROOT}\usr\share\RuntimeMergeModules" -p:SWIFT_LINT_BUILD=${{ github.workspace }}\BinaryCache\SwiftLint\x86_64-unknown-windows-msvc\release -p:OutputPath=${{ github.workspace }}\BinaryCache\artifacts -p:RunWixToolsOutOfProc=true
+        run: msbuild ${{ github.workspace }}/SourceCache/swift-build/platforms/Windows/SwiftLint.wixproj -nologo -restore -p:Configuration=Release -p:RedistributablesDirectory="${env:SDKROOT}\..\..\..\..\..\..\Redistributables\0.0.0" -p:SWIFT_LINT_BUILD=${{ github.workspace }}\BinaryCache\SwiftLint\x86_64-unknown-windows-msvc\release -p:OutputPath=${{ github.workspace }}\BinaryCache\artifacts -p:RunWixToolsOutOfProc=true
```

Comment:
That's quite the relative path lol. I assume the directory structure there is stable?

Reply:
Stable enough for now, I need to figure out a way to query this in the future.